include/mbedtls/ecp.h

```diff
@@ -31,13 +43,13 @@
  */
 #define MBEDTLS_ERR_ECP_BAD_INPUT_DATA                    -0x4F80  /**< Bad input parameters to function. */
 #define MBEDTLS_ERR_ECP_BUFFER_TOO_SMALL                  -0x4F00  /**< The buffer is too small to write to. */
-#define MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE               -0x4E80  /**< Requested curve not available. */
+#define MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE               -0x4E80  /**< The requested curve is not available. */
```

Comment:
Pre-existing: This comment is not accurate, there are other occasions when this error code is returned. Suggestion: `The requested feature (e.g. the support for a specific curve or for a format of point compression) is not available.`

Reply:
@RosZad Sounds good to me.